qemu/tests/cfg/cpu_model_negative.cfg

```diff
@@ -10,6 +11,9 @@
         - smp_lt_maxcpu:
             wrong_cmd = "-smp 8,maxcpus=4,cores=2,threads=2,dies=1,sockets=2"
             warning_msg = 'maxcpus must be equal to or greater than smp'
+                required_qemu = [, 6.2.0)
+            warning_msg = 'qemu-kvm: Invalid CPU topology: product of the hierarchy must match maxcpus'
```

Comment:
`warning_msg` is not a valid variant, so indent is not applied, the second `warning_msg` and `required_qemu` will overwrite the first one.

Reply:
> `warning_msg` is not a valid variant, so indent is not applied, the second `warning_msg` and `required_qemu` will overwrite the first one.

update, and thanks.